zerver/openapi/python_examples.py

```diff
@@ -314,6 +314,26 @@ def get_subscription_status(client: Client) -> None:
     )
 
 
+@openapi_test_function("/invites:post")
+def send_invites(client: Client) -> None:
+
+    # {code_example|start}
+    # Create the request with email and stream IDs
+    request = {"invitee_emails": "bob-test@zulip.com", "invite_as": 100, "stream_ids": [1]}
```

Comment:
It's best to use `@example.com` for any example emails, especially in content that'll be on the web.

Reply:
Can you extract `ROLE_MEMBER = 400` as a constant and use it here?  

Actually, to make this code reasonable, we should probably add a commit to `python-zulip-api` to make a `client.constants.ROLE_MEMBER` set of properties that's available in the Python bindings for any code that wants to use them.  Let me start a thread about this.

Reply:
Discussion started in https://chat.zulip.org/#narrow/stream/378-api-design/topic/python.20bindings.20and.20constants/near/1240760.